### PR TITLE
POLIO-1575: add age fields to round

### PR DESCRIPTION
**Pull request by quang-le**
Age fierlds missing in round form

Related JIRA tickets : POLIO-1575

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [x] New translations have been added or updated if new strings have been introduced in the frontend
- [x] My migrations file are included
- [x] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Doc
N/A

## Changes

- Add fields to `Round`model
- Add input fields to `RoundForm`

## How to test

- Go to rounds tab, try to add fields and save

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/5ebda8f1-c68a-43c0-aa19-3caa5161a3c8

### CODE REVIEW

File: plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx
Comment (beygorghor) [inline plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx:1]: we can also validate thahthe max is not below the min i think
Screenshot 2024-06-24 at 12 18 24

**Review by beygorghor — CHANGES_REQUESTED** [review]

File: plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx
Comment (quang-le) [reply plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx:1]: Ah yeah, true, I forgot

Comment (quang-le) [reply plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx:1]: Uploading Screen Recording 2024-06-24 at 13.01.07.mov…

Comment (quang-le) [reply plugins/polio/js/src/domains/Campaigns/Rounds/RoundForm.tsx:1]: https://github.com/BLSQ/iaso/assets/38907762/48af5354-f02d-497d-aaa8-89cdc1c045b3

**Review by beygorghor — APPROVED** [review]

**Pull request merged**